chain/gen/genesis/miners.go

```diff
@@ -135,14 +138,32 @@ func SetupStorageMiners(ctx context.Context, cs *store.ChainStore, sys vm.Syscal
 		}
 
 		{
-			constructorParams := &power0.CreateMinerParams{
-				Owner:         m.Worker,
-				Worker:        m.Worker,
-				Peer:          []byte(m.PeerId),
-				SealProofType: spt,
+			var params []byte
+			if nv <= network.Version10 {
```

Comment:
This change was unearthed, but not caused by FIP-0061 -- it was "working" before because the `SealProofType` was coincidentally the same value as the `PoStProofType`, and the `power2.CreateMinerParams` marshaled to the same bytes as the `power3.CreateMinerParams`, and so it...worked.